reflectometry/output.ort

```diff
@@ -44,202 +44,51 @@
 #   unit: dimensionless
 #   comment: fractional description of sigma from Gaussian resolution function
 # 
-8.3618090452261314e-03 1.1339954137802124e+00 1.3041149079799652e-01 3.8581113644695630e-02
```

Comment:
Why have the results changed? because the bins are different?
it suggests this test is going to break again in the future, and if we are always overwriting the test results, it may suggest we are not really testing much?

Reply:
Looks like there are much fewer bins to me...

Reply:
> Looks like there are much fewer bins to me...

Are there meant to be fewer bins or is this a bug?

Reply:
I think it is a result of a change to the `q_grid` function?

Reply:
So you are writing this file as part of the notebook, but it is not being used after that.
So it doesn't need to be under version control, right?
Can we remove it from the codebase?

Reply:
It is linked in the notebook text so the user can download and have a peek